app/services/flash-messages.js

```diff
@@ -1 +1,12 @@
-export { default } from 'ember-cli-flash/services/flash-messages';
+import FlashMessagesService from 'ember-cli-flash/services/flash-messages';
+import { computed } from '@ember/object';
+import flashMessageOptions from 'ember-cli-flash/utils/flash-message-options';
+
+// import the application's config
+import config from '../config/environment';
```

Comment:
I wonder if we need to do something a little more robust like: `getOwner(this).resolveRegistration('config:environment');` to get the config?